pallets/subtensor/src/staking.rs

```diff
@@ -190,11 +191,7 @@ impl<T: Config> Pallet<T> {
         Self::set_last_tx_block(&coldkey, block);
 
         // --- 10. Emit the staking event.
-        Self::set_stakes_this_interval_for_hotkey(
-            &hotkey,
-            stakes_this_interval + 1,
-            block,
-        );
+        Self::set_stakes_this_interval_for_hotkey(&hotkey, stakes_this_interval + 1, block);
```

Comment:
we definitely want format on save, pretty much everything it does will be CI enforced soon if it isn't already

Reply:
In the past , formatOnSave has really screwed me over , especially when your settings are different from the repo , you loads of changes in PRs which are just formatting issues and lets not even mention rebases. 

I think it might better to have a make / just file and people run make/ just lint before every commit , so that its standardised

Reply:
https://discord.com/channels/799672011265015819/1176889593136693339/1229443862909423697